pkgs/by-name/gs/gssg/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  fetchFromSourcehut,
+  buildGoModule,
+}:
+
+buildGoModule {
+  pname = "gssg";
+  version = "0-unstable-2024-11-23";
```

Comment:
Please use the date of the git commit for the version. See https://github.com/NixOS/nixpkgs/tree/24a2206ec1be5ecb32acf8ab089333eb75d7c4b1/pkgs#versioning.